RegistrationWindow/RegistrationWindow/VIew/RegistrationController.swift

```diff
@@ -0,0 +1,105 @@
+import UIKit
+
+class RegistController: UIViewController {
+    private var textfieldName = UITextField()
+    private var textfieldPassword = UITextField()
+    private var acceptButton = UIButton()
+    private var infoLabel = UILabel()
+    var viewModel: RegistrationViewModel?
+    var coordinator: Coordinator?
+    
+    init(viewModel: RegistrationViewModel) {
+        self.viewModel = viewModel
+        super.init(nibName: nil, bundle: nil)
+    }
+    
+    required init?(coder: NSCoder) {
+        super.init(coder: coder)
+    }
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setupSubview()
```

Comment:
create protocol like 'ControlSetup' to store these functions there and implement this protocol to your viewControllers